src/screens/Article.js

```diff
@@ -1,87 +1,98 @@
 import theme from '../theme';
 
-const bgImage = 'https://images.unsplash.com/photo-1533106497176-45ae19e68ba2?fit=crop&w=760&q=80';
+const { width, height } = Dimensions.get('screen');
+
+const bgImage = 'https://images.unsplash.com/photo-1516651029879-bcd191e7d33b?fit=crop&w=330&q=80';
 
 const Article = props => (
-  <ImageBackground source={{ uri: bgImage }} style={{ width: '100%', height: '100%' }}>
-    <Block flex style={styles.bottom}>
-      <StatusBar hidden={false} barStyle="light-content" />
-      <Block style={styles.navbar}>
-        <NavBar transparent leftIconColor={theme.COLORS.WHITE} onLeftPress={() => props.navigation.openDrawer()} />
-      </Block>
+  <Block>
+    <StatusBar hidden={false} barStyle="light-content" />
+    <Block style={styles.navbar}>
+      <NavBar transparent leftIconColor={theme.COLORS.WHITE} onLeftPress={() => props.navigation.openDrawer()} />
+    </Block>
 
-      <Block style={styles.articleContainer}>
-        <Block flex style={{ marginBottom: theme.SIZES.BASE }}>
+    <Image source={{ uri: bgImage }} resizeMode="cover" style={{ width, height: height * 0.55 }} />
+
+    <Block center style={{ marginTop: -theme.SIZES.BASE * 2 }}>
+      <Block flex style={styles.header}>
+        <Block>
           <Text h3>I would happily watch a TV show about crabs</Text>
-          <Text muted size={theme.SIZES.FONT} style={{ marginTop: theme.SIZES.BASE, fontWeight: '600' }}>
+          <Text muted style={{ marginTop: theme.SIZES.BASE, fontWeight: '500' }}>
             InterBlocking this super star
           </Text>
         </Block>
-        <Block flex>
-          <AuthorSection
-            imageSource="http://i.pravatar.cc/100"
-            title="Alin Gheorghe"
-            subTitle="420 minutes ago"
-            style={{ paddingHorizontal: 0 }}
-          />
+        <Block space="between" row style={styles.stats}>
+          <Block flex={0.25}>
+            <Image source={{ uri: 'http://i.pravatar.cc/100' }} style={styles.avatar} />
+          </Block>
+          <Block flex={0.7} style={styles.middle}>
+            <Text style={{ fontWeight: '500' }}>Galio Framework</Text>
+            <Text p muted>16 minutes ago</Text>
+          </Block>
+          <Block flex={0.5} row middle space="around">
+            <Block row middle>
+              <Icon name="eye" family="MaterialCommunityIcons" color={theme.COLORS.MUTED} size={theme.SIZES.FONT * 0.8} />
+              <Text p muted style={{ marginLeft: theme.SIZES.BASE * 0.25 }}>25.6k</Text>
+            </Block>
+            <Block row middle>
+              <Icon name="heart-outline" family="MaterialCommunityIcons" color={theme.COLORS.MUTED} size={theme.SIZES.FONT * 0.8} />
+              <Text p muted style={{ marginLeft: theme.SIZES.BASE * 0.25 }}>936</Text>
+            </Block>
+          </Block>
         </Block>
-        <Block flex>
-          <Text style={{ marginTop: 10, lineHeight: theme.SIZES.FONT * 1.4 }}>
+        <ScrollView>
```

Comment:
ScrollView is not needed here even though it doesn't affect the looks or the behavior of our screen. The main concept was for the whole white Block to move up with the finger in order to hide the image underneath. Something like parallax.

Reply:
Will be implemented in v2 because of low specs on this.